src/test/java/com/sugarcrm/candybean/candybeanRunner/CandybeanRunnerSystemTest.java

```diff
@@ -0,0 +1,102 @@
+package com.sugarcrm.candybean.candybeanRunner;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.*;
+import org.junit.runner.RunWith;
+
+import java.io.*;
+
+/**
+ * A series of test intended to test some of the expected function
+ * Since there doesn't appear to be a good way to test a runner, these
+ * tests certainly are not conclusive.
+ */
+@RunWith(CandybeanRunner.class)
+public class CandybeanRunnerSystemTest {
+	static String filename = "./candybeanTest.txt";
+
+	@BeforeClass
+	static public void createFile() {
+		File f = new File(filename);
+		if (f.exists()) {
+			if (!f.delete()) {
+				Assert.fail("Could not delete external resource " + filename);
+			}
+		} else {
+			try {
+				if (!f.createNewFile()) {
+					Assert.fail("Could not create external resource " + filename);
+				}
+			} catch (IOException e) {
+				Assert.fail("Could not create external resource " + filename);
+			}
+		}
```

Comment:
Does this really want to be in an else?  If the file did exist and we deleted it, don't we *still* want to create it afterward as we would have had it not existed?